index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
I think the one in `public/index.html` can be removed then if the index.html is in the root folder and was AFAIK specific to CRA behavior.

Reply:
Yeah, I haven't gotten there yet. I just got the code compiling and wanted to put something up.

My favorite part about the PR is the -12k lines in `yarn.lock`

Reply:
React-scripts is a pretty heavy one on the dependency side: https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/package.json#L30 almost utterly unbelievable.